### DISCUSSION TITLE
Multi-Columns For an Index

### DISCUSSION THREAD

**Discussion by ahadvirus** [fluentmigrator/fluentmigrator#1659] [Q&A]
Hi there.

I am new in the community, and trying to use a new thing for ORM, so I switch from EF Core to Dapper, then I realize the Dapper doesn't have a migration system by default, so I find the package.

You could choose multi-columns at EF Core, for example, `builder.HasIndex(x => new { x.Name, x.UserId}).IsUnique()`.
After I read the documentation, I got how to create an index, but I couldn't choose multi-columns for an index.

Please help me to figure out how to choose multi-columns for an index.

**Comment by pmarflee** [comment]
To create an index that spans multiple columns, chain together calls to `OnColumn`, specifying the direction for each field contained within the index. For example:

```            
Create.Index("IX_MyIndex").OnTable("MyTable")
  .OnColumn("Column1").Ascending()
  .OnColumn("Column2").Ascending()
  .OnColumn("Column3").Ascending();
```
